servers/cromwell/jobs/swagger/swagger.yaml

```diff
@@ -371,25 +371,28 @@ definitions:
     properties:
       userId:
         type: "string"
-        description: "The user associated with the job"
+        description: "The user associated with the job."
       statusDetail:
         type: "string"
         description: "Longer text description of the job status."
       logs:
         type: "object"
-        description: "Map of type of log file to its file location"
+        description: "Map of type of log file to its file location."
         properties: {}
       lastUpdate:
         type: "string"
         format: "date-time"
-        description: "Job last update datetime in ISO8601 format"
+        description: "Job last update datetime in ISO8601 format."
       envs:
         type: "object"
-        description: "Map of ENV variables associated with the job"
+        description: "Map of ENV variables key values associated with the job."
         properties: {}
+      script:
```

Comment:
Per offline discussion, perhaps consider making this something more generic such as `sourceFile`? This could then more easily be be used to, for example, display a Cromwell WDL file; even though I guess we don't have a way of displaying that currently.